include/ccf/receipt.h

```diff
@@ -35,13 +35,21 @@ namespace ccf
       bool operator==(const LeafComponents& other) const = default;
     };
 
+    /// Signature over the root of the Merkle Tree, by the node private key
     std::string signature;
+    /// Root of the Merkle Tree
     std::optional<std::string> root = std::nullopt;
```

Comment:
Is still included in the current receipts or only there for 1.x compatibility?

Reply:
Only there for compatibility I believe, but will check.

Reply:
yes